src/Settings/PostGrid.php

```diff
@@ -189,15 +189,17 @@ public static function filterBar($settings): string
         $show_filter = false;
         $tax_exists = false;
 
-        $postType = $settings->post_type ?? null;
+        $postTypes = (array) ($settings->post_type ?? ['post']);
         if ('main_query' === $settings->data_source) {
-            $postType = get_post_type();
+            $postTypes = [get_post_type()];
         }
-        if (empty($postType)) {
-            $postType = 'post';
+        if (empty($postTypes)) {
+            $postTypes = ['post'];
         }
```

Comment:
`get_post_type()` can return `false`.

```shell
php > var_dump(empty([false]));
bool(false)
```

Reply:
Rufua is alive 😲

Reply:
Good thing I check for false :) hello pal!

Reply:
Oh, i see line 202 now.
I am stupid as always

Reply:
`reset` can also return `false` :) double-check!

hope you are doing well mate